service/service.go

```diff
@@ -622,7 +623,8 @@ func (s *service) getNewIsilonConfigs(ctx context.Context, configBytes []byte) (
 	}
 
 	newIsiClusters := make(map[interface{}]interface{})
-	for i, config := range inputConfigs.IsilonClusters {
+	for i, clusterConfig := range inputConfigs.IsilonClusters {
+		config := clusterConfig
```

Comment:
why introduce one more variable? either use config or clusterConfig, and remove one of these.

Reply:
This is to address the below,
[/github/workspace/service/service.go:664] - G601 (CWE-118): Implicit memory aliasing in for loop. (Confidence: MEDIUM, Severity: MEDIUM)
  > 664: 		config.isiSvc, _ = s.GetIsiService(clientCtx, &config, logLevel)